plugins/obs-filters/cross-stitching-filter.c

```diff
@@ -0,0 +1,259 @@
+/*****************************************************************************
+Copyright (C) 2016 by c3r1c3 <c3r1c3@nevermindonline.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*****************************************************************************/
+
+#include <obs-module.h>
+
+#define SETTING_STITCH_GAP            "stitch_gap"
+#define SETTING_NEGATIVE_BIAS         "negative_bias"
+#define SETTING_STITCH_COLOR          "stitch_color"
+#define SETTING_OPACITY               "stitch_opacity"
+#define SETTING_INVERT                "invert"
+#define SETTING_GREYSCALE             "greyscale"
+
+#define OMT                           obs_module_text
+#define TEXT_STITCH_GAP               OMT("Stitch.Gap")
+#define TEXT_NEGATIVE_BIAS            OMT("Negative.Bias")
+#define TEXT_STITCH_COLOR             OMT("Stitch.Color")
+#define TEXT_OPACITY                  OMT("Stitch.Opacity")
+#define TEXT_INVERT                   OMT("Invert.Stitching")
+#define TEXT_GREYSCALE                OMT("Greyscale")
+
+
+struct cross_stitching_filter_data {
+	obs_source_t                   *context;
+
+	gs_effect_t                    *effect;
+
+	gs_eparam_t                    *stitch_gap_param;
+	gs_eparam_t                    *negative_bias_param;
+	gs_eparam_t                    *stitch_color_param;
+	gs_eparam_t                    *invert_param;
+	gs_eparam_t                    *greyscale_param;
+
+	uint8_t                         stitch_gap;
+	uint8_t                         negative_bias;
+	struct vec4                     stitch_color;
+	bool                            invert;
+	bool                            greyscale;
+};
+
+
+/*
+ * As the functions' namesake, this provides the user facing name
```

Comment:
I meant about this particular comment attached to this particular function.  If this comment is describing this single function's purpose, then it would make more sense to be worded:

> As the function's namesake, this provides the user facing name

As I'm rereading this though, using "namesake" doesn't really make sense either, since "namesake" is typically used when referring to a person who is named after another person (or the person from whom another's name originates).  It would make more sense altogether to use this:

> As the function's name implies, this provides the user facing name